### ISSUE TITLE
toolrunner.arg adds unwanted quotes around value

### ISSUE THREAD

**Issue by scott-lin** [Microsoft/vsts-task-lib#312]
**tldr**
From what I can tell, when an argument value contains a space, [toolrunner.arg](https://github.com/Microsoft/vsts-task-lib/blob/master/node/docs/vsts-task-lib.md#toolrunnerToolRunnerarg) adds double-quotes around the value automatically. In my situation, I cannot have these quotes present as my command is not valid with them.

**Full details**
If there's a workaround to my situation below, I'm all ears.

Command I need to execute:
- I am calling a script and then bubbling up its exit code, so vsts-task-lib can capture the return code.
- I am forced to use `-Command` of powershell.exe because one of the arguments to the script is an array. The `-File` parameter of powershell.exe does not support arrays [it seems](https://stackoverflow.com/a/33230805/3897362), so I cannot use it.
- The quotes look funny, but I've tested this command and it works. You can do more reading about the quotes [here](https://stackoverflow.com/a/28722559/3897362).

```
powershell.exe -NoLogo -NoProfile -NonInteractive -Command '&{MyScript.ps1 -InputPath ''some\path'' -OutputPath ''some\path2'' -VisualStudioXmlDocumentationPath ''some\path\Foo 1.xml'' -AssemblyPaths ''some\path\Foo 1.dll'',''some\path\Foo 2.dll'',''some\path\Foo 3.dll'' -Version 3 Format JSON -TreatGenerationWarningAsError}; EXIT $LASTEXITCODE'
```

In order to build this command, I'm doing something like this:

```
tl.tool(tl.which('PowerShell', true))
    .arg('-NoLogo')
    .arg('-NoProfile')
    .arg('-NonInteractive')
    .arg('-Command')
    .arg('<my large -Command argument requiring no double-quotes, but gets them anyway due to spaces being present>');
```

The above code results in an invalid command:

```
powershell.exe -NoLogo -NoProfile -NonInteractive -Command "'&{MyScript.ps1 -InputPath ''some\path'' -OutputPath ''some\path2'' -VisualStudioXmlDocumentationPath ''some\path\Foo 1.xml'' -AssemblyPaths ''some\path\Foo 1.dll'',''some\path\Foo 2.dll'',''some\path\Foo 3.dll'' -Version 3 Format JSON -TreatGenerationWarningAsError}; EXIT $LASTEXITCODE'"
```

**Comment by ericsciple** [comment]
Use the `windowsVerbatimArguments` option https://github.com/Microsoft/vsts-task-lib/blob/master/node/toolrunner.ts#L40

**Comment by scott-lin** [comment]
Thank you, works well. Would love to see this added to the [TypeScript API reference](https://github.com/Microsoft/vsts-task-lib/blob/master/node/docs/vsts-task-lib.md).
